### DISCUSSION TITLE
Re: effect/other allele

### DISCUSSION THREAD

**Discussion by kentaro0209** [PGScatalog/pgsc_calc#258] [Q&A] [answered]
Hello, 

I'm calculating the PGS of the genome input files (bfiles) from our GWAS data with the PGS002814 score file.
I just found out some of the effect/other allele on the PGS002814 score file and our cohort's bim file are opposite as below.
 
 PGS002814 score file
 chr_name	chr_position	effect_allele	other_allele	effect_weight
 1	854250	G	A	0.0325005
 
 Our bim file
 1	1:854250	0	854250	A	G
 
 Is it possible that the score was summed up in the opposite direction? If so, is there any way to adjust this?
 I got the PGS score which is the opposite result as I expected, so I'm wondering why this happened.

**Comment by smlmbrt** [comment, marked as answer]
The difference in REF/ALT orientation shouldn't matter in this case, pgsc_calc (plink underlying) will count the dosage of the effect allele G and use it to calculate the score. It is sometimes possible that the authors provide the PGS Catalog scoring files where the alleles are swapped or weights reversed which can yield a PRS that is anti-correlated with your trait, it might simply be that.
